doc/rst/usingchapel/chplenv.rst

```diff
@@ -508,6 +508,28 @@ CHPL_HWLOC
    to ``none`` and rebuild (and please file a bug with the Chapel team.) Note
    that building without hwloc will have a negative impact on performance.
 
+.. _readme-chplenv.CHPL_JEMALLOC:
+
+CHPL_JEMALLOC
+~~~~~~~~~~~~~
+   Optionally, the ``CHPL_JEMALLOC`` environment variable can select
+   between no jemalloc, using the jemalloc distributed with Chapel in
+   third-party, or using a system jemalloc. This setting is intented
+   to elaborate upon ``CHPL_MEM=jemalloc``.
+
+       ======== ==============================================================
+       Value    Description
+       ======== ==============================================================
+       none     do not build or use jemalloc
```

Comment:
Perhaps one day it will be. For example, we could start building the compiler with jemalloc. Doing so gives a 10% performance boost, in my measurements.